src/sourcemaps/command.js

```diff
@@ -50,13 +58,32 @@ exports.handler = async function (argv) {
 
   await scanner.scan();
 
-  const uploader = new Uploader({
-    accessToken: argv['access-token'],
-    baseUrl: argv['url-prefix'],
-    codeVersion: argv['code-version']
-  })
+  if (argv['signed-url']) {
+    const requester = new Requester({
+      accessToken: argv['access-token'],
+      baseUrl: argv['url-prefix'],
+      codeVersion: argv['code-version']
+    })
+
+    const signedUrlUploader = new SignedUrlUploader()
+    await requester.requestSignedUrl(argv['dry-run'])
+    if (requester.data && requester.data['err'] === 0) {
+      requester.setProjectID(requester.data['result']['project_id'])
+      const manifestFile = requester.createManifestFile(scanner.targetPath)
+      signedUrlUploader.mapFiles(scanner.files);
+      signedUrlUploader.zipFiles(scanner.targetPath, requester.data['result']['filename'], manifestFile);
+      await signedUrlUploader.upload(argv['dry-run'], requester.data['result']['signed_url'])
```

Comment:
This is a lot of logic of the requester and uploader present in command.js. Ideally, this module should just route the arguments to the correct class to do its work internally. 

Maybe for example, the requester instance can be passed to the `SignedUrlUploader` constructor:
```
const signedUrlUploader = new SignedUrlUploader(requester);
```
Then all of the remaining logic could be encapsulated in something like:
```
signedUrlUploader.upload();
```
There are other reasonable ways, but the idea is to get the logic into the relevant classes.